addons/gamification/security/ir.model.access.csv

```diff
@@ -2,18 +2,26 @@ id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
 
 goal_employee,"Goal Employee",model_gamification_goal,base.group_user,1,1,0,0
 goal_manager,"Goal Manager",model_gamification_goal,base.group_erp_manager,1,1,1,1
+goal_portal,"Goal Portal",gamification.model_gamification_goal,base.group_portal,1,1,0,0
```

Comment:
I agree, but these are "move" commit, so I just copy/paste code without any change #tracability. Should I rename the rule anyway ?